02/app.js

```diff
@@ -9,20 +9,29 @@ function init() {
 
     initEventWithError(clickEl, 'click', new RangeError('Błąd zakresu!'));
     initEventWithError(enterEl, 'mouseenter', new TypeError('Błąd typu!'));
-
 }
 
 function setRandomPosition(element, error = null) {
     element.style.top = Math.random() * 600 + 'px';
     element.style.left = Math.random() * 800 + 'px';
 
+    try {
     if(error) {
         throw error;
     }
+    } catch (e) {
+        if (e instanceof TypeError) {
+        document.querySelector('.alert').classList.remove('alert--hidden')
+        document.querySelector('.alert__message').innerText = 'Błąd typu!'
+    } 
+      else if (e instanceof RangeError) {
+        document.querySelector('.alert').classList.remove('alert--hidden')
+        document.querySelector('.alert__message').innerText = 'Błąd zakresu!'
+    }} 
 }
 
 function initEventWithError(element, eventName, error) {
     element.addEventListener(eventName, function() {
         setRandomPosition(this, error);
```

Comment:
To tutaj powinno być przechwycenie błędu. To ta metoda rzuca błędem i tutaj najwcześniej to przechwytujemy.
Robienie tego wew. tej funkcji to po prostu specjalna forma if-a. W takim wypadku lepiej użyć normalnego if-a.